### PR TITLE
Gravitational Pushsplits

### PR DESCRIPTION
**Pull request by Agarian**
This is just @AlexHGaming's PR's merged into one (and revised).

### CODE REVIEW

**Comment by GhostShot3** [comment]
What is pushsplit ?

**Comment by Agarian** [comment]
@GhostShot3 see the beginning of [this video](https://www.youtube.com/watch?v=PB0s5wrXSRs) to see what this PR is referencing. There's actually two types, but this one has been nicknamed the "gravitational pushsplit".

**Comment by AlexHGaming** [comment]
@Agarian I swear increasing it makes it harder.

**Comment by Agarian** [comment]
Well you see, thats kind of why I only wwnted it to be enabled or disabled, because I noticed that too...

**Comment by GhostShot3** [comment]
Looks like a bug

File: src/gameserver.ini [hunk 1]
```diff
@@ -170,6 +172,7 @@ playerRecombineTime = 30
 playerMaxNickLength = 15
 playerDisconnectTime = -1
 splitVelocity = 780
+gravitationalPushsplit = 0
```
Comment (fayizan) [inline src/gameserver.ini]: "gravitationalPushsplits = 0"

**Pull request closed without merging**